SlicerCART/documentation/installation.md

```diff
@@ -0,0 +1,56 @@
+# SlicerCART Installation
+
+### Requirements
+
+* MacOS Sonoma or Sequoia is recommended 
+* A working version of [3D Slicer](https://download.slicer.org).
+  * N.B. The version currently used to develop this module is the version 5.6.2 (most stable release as of 2024-10-22). The version 5.2.2 has also been used.
+
+This module is currently used on:
+
+* MacOS Sequoia 15.0.1
+* 3D Slicer version 5.6.2
+
+Although it may work on other versions and/or operating system, please note that it has not been tested.
+
+## Installation Steps
+
+1. Install [3D Slicer](https://download.slicer.org).
+2. Make sure that you are able to open and use the 3D Slicer software before 
+   trying installing any extension/module. 
+   3. If you encounter some problems, you are encouraged to refer to:
+      4. [3D Slicer Documentation](https://slicer.readthedocs.io/en/latest/)
+      5. [3D Slicer forum](https://discourse.slicer.org/) (very active community)
```

Comment:
What do the numbers refer to?

Reply:
The steps in order (e.g. 1 before 2, 2 before 3, etc.) Like a recipe, just follow the steps?

Reply:
this is how it looks:

![image](https://github.com/user-attachments/assets/0374d35b-ae7c-49ad-aaca-ee0c6e1a2e7d)


inside the step 2: there is 3, 4 and 5, but then it goes again back to 3 under, this is confusing, please fix